config/routes.rb

```diff
@@ -39,7 +40,8 @@
 
   get "search/", to: "search#search"
 
-  resources :tickets, only: %i[new create]
+  get "tickets/new", to: redirect("https://www.data.gov.uk/support")
```

Comment:
```suggestion
  get "tickets/new", to: redirect("/support")
```

I think that's correct 🙏